enterprise/server/remote_execution/container/container.go

```diff
@@ -335,9 +338,18 @@ func (s *UsageStats) TaskStats() *repb.UsageStats {
 }
 
 func (s *UsageStats) updateTimeline(now time.Time) {
-	if 8*(len(s.timeline.GetCpuSamples())+len(s.timeline.GetTimestamps())) > timeseriesSizeLimitBytes {
+	totalLength := 0
```

Comment:
nit: this loop is neat, but this is shorter and probably easier to read:
```
st := s.timeline
totalLength := st.GetTimestamps() + st.GetCpuSample() + st.GetMemoryKbSamples()
```

Reply:
Done